application/client/src/app/service/session/dependencies/search/charts/store.ts

```diff
@@ -1,13 +1,22 @@
 import { Key, Store, StoredEntity } from '../store';
 import { ChartRequest } from './request';
 import { IFilter } from '@platform/types/filter';
-import { DisableConvertable } from '../disabled/converting';
-
+import { Subject } from '@platform/env/subscription';
+import { ISelectEvent } from '@ui/views/sidebar/search/providers/definitions/provider';
```

Comment:
service should not refer to any UI component directly, it's break a logic of service (as independent on UI entity)